scripts/delete-cluster-projects.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+shout() {
+   set +x
+  echo -e "\n.---------------------------------------\n${1}\n'---------------------------------------\n"
+   set -x
+}
+
+set -ex
+
+shout "| Cleaning up projects from cluster ..."
+
+shout "| Logging in to minishift..."
+
+oc login -u developer -p developer --insecure-skip-tls-verify $(minishift ip):8443
+PROJECT_LIST=$(oc get projects)
+
+for i in $PROJECT_LIST; do
```

Comment:
Why is this needed?

Reply:
This was to cleanup any projects left behind in the cluster, however I will remove it for now as we can approach the "no proper cleanup by tests" problem in a better way.